st2common/st2common/util/param.py

```diff
@@ -150,8 +152,32 @@ def _render(node, render_context):
     Render the node depending on its type
     '''
     if 'template' in node:
+        complex_type = False
+
+        LOG.debug("Rendering param of type: %s", type(node['template']))
```

Comment:
Meant to take this out. I added it in when I was debugging why context was getting rendered incorrectly